### PR TITLE
Separate liveData requests for faster page load

### PR DESCRIPTION
**Pull request by echaidemenos**
https://github.com/aqualinkorg/aqualink-app/issues/703

Splits `liveData` from `Site`, allowing components that don't need the the result of the time demanding liveData request to render.

### CODE REVIEW

File: packages/website/src/common/SiteDetails/index.tsx [hunk 1]
```diff
@@ -44,33 +49,36 @@ const SiteDetails = ({
 }: SiteDetailsProps) => {
   const classes = useStyles();
   const theme = useTheme();
+  const liveData = useSelector(liveDataSelector);
+  const liveDataLoading = useSelector(siteLiveDataLoadingSelector);
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/index.tsx:53]: could we just assume that the data is loading while `liveData` it's undefined?

File: packages/website/src/common/SiteDetails/index.tsx [hunk 2]
```diff
@@ -185,7 +193,11 @@ const SiteDetails = ({
         {cards.map((Component, index) => (
           <Grid key={index.toString()} item xs={12} sm={6} md={3}>
             <div className={classes.card}>
-              <LoadingSkeleton variant="rect" height="100%" loading={isLoading}>
+              <LoadingSkeleton
+                variant="rect"
+                height="100%"
+                loading={isLoading || liveDataLoading}
```
Comment (ericboucher) [inline packages/website/src/common/SiteDetails/index.tsx:199]: won't this make all the components wait until liveData has loaded?

Comment (ericboucher) [reply packages/website/src/common/SiteDetails/index.tsx:199]: It's ok if they all depend on the liveData but I am not sure that's the case?

Comment (echaidemenos) [reply packages/website/src/common/SiteDetails/index.tsx]: these skeletons are for the 4 cards under the map. They all depend on `liveData`
https://github.com/aqualinkorg/aqualink-app/blob/4ab6418c03fbc18ebef2405168ee7351c704c190/packages/website/src/common/SiteDetails/index.tsx#L60-L81

Comment (ericboucher) [reply packages/website/src/common/SiteDetails/index.tsx]: Indeed, thanks for checking

File: packages/website/src/common/SiteDetails/index.tsx (hunk 1 above)
Comment (ericboucher) [reply packages/website/src/common/SiteDetails/index.tsx]: This works ok, but we are having the same problem as #709 I think. We need to make sure to reset these values to undefined.

As it is know, navigating to a new site keeps the cards up with the value from the site you are navigating from, which is not ideal.

**Comment by echaidemenos** [comment]
this commit will also resolve https://github.com/aqualinkorg/aqualink-app/issues/709

**Review by ericboucher — APPROVED** [review]

**Pull request merged**